rocAL/include/api/rocal_api_types.h

```diff
@@ -377,4 +377,17 @@ enum RocalExternalSourceMode {
     ROCAL_EXTSOURCE_RAW_UNCOMPRESSED = 2,
 };
 
+/*! \brief Tensor Last Batch Policies
+ *  \ingroup group_rocal_types
+ The last batch policies determine the behavior when there are not enough samples in the epoch to fill the last batch
+        FILL - The last batch is filled by either repeating the last sample or by wrapping up the data set.
```

Comment:
Better to add these one-liner descriptions below for each policy. Remove the exaplanation from here.